pkgs/development/python-modules/wsproto/default.nix

```diff
@@ -1,15 +1,19 @@
-{ lib, buildPythonPackage, fetchPypi, h11, enum34, pytest }:
+{ lib, buildPythonPackage, fetchPypi, isPy27
+, h11
+, pytest
+}:
 
 buildPythonPackage rec {
   pname = "wsproto";
-  version = "0.14.1";
+  version = "0.15.0";
+  disabled = isPy27;
```

Comment:
Apparently, python-3.5 support is removed in release 0.15.0, and we have a `python35Packages` packages-set in `nixpkgs` (see https://github.com/python-hyper/wsproto/blob/master/CHANGELOG.rst#0150-2019-08-10).

I think we should prefer something like `disabled = pythonOlder "3.6"`. And of course, the same goes in `python-packages.nix`.

Reply:
you're correct sir, updating :)

Reply:
updated, 2.7 && 3.5 -> 0.14.1 and 3.6 && 3.7 -> 0.15.0